### PR TITLE
Add programme for Day 2 of 2018 Users meeting

### PR DESCRIPTION
**Pull request by sbesson**
Couple of initial decisions open for discussion in addition to the usual spelling/formatting review:

- I re-used the same CSS layout as the [previous Users meeting](https://www.openmicroscopy.org/events/12th-annual-users-meeting-2017.html)
- for the morning workshops, the `event-title` class is used to highlight the workshop theme
- the program only includes workshops titles for now. Bullet points can be added in a second step or once final
- the last workshop is expected to grow as we receive interest from the community

### CODE REVIEW

**Comment by mtbc** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/13th-annual-users-meeting-2018.html.

File: events/13th-annual-users-meeting-2018.html [hunk 1]
```diff
@@ -206,6 +206,129 @@ <h2 class="event-day">Day 2 — Thursday 31st May, 2018 - Dalhousie Teaching Roo
...
+                <ul class="event-workshop">
+                    <li class="event-workshop-title">
+                    <span class="event-workshop-room">Room 1</span> -
+                    Analyze raw data with OMERO</li>
+                </ul>
+            </div>
+        </div>
+        <div class="event-row row">
+            <div class="small-12 medium-2 columns">
+                <h5 class="event-time">11:00-11:30</h5>
```
Comment (mtbc) [inline events/13th-annual-users-meeting-2018.html]: spaces around time range hyphens

**Comment by sbesson** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/events/13th-annual-users-meeting-2018.html for review

**Comment by will-moore** [comment]
Looks good 👍

**Comment by jburel** [comment]
R notebook has been moved to handle metadata

File: events/13th-annual-users-meeting-2018.html [hunk 2]
```diff
@@ -229,6 +229,164 @@ <h2 class="event-day">Day 2 — Thursday 31st May, 2018 - Dalhousie Teaching Roo
...
+                <h5 class="event-time">09:00 - 10:00</h5>
+            </div>
+            <div class="small-12 medium-10 columns">
+                <h3 class="event-title">Workshop</h3>
+                <ul class="event-workshop">
+                    <li class="event-workshop-title">
+                    <span class="event-workshop-room">Room 1</span> - 
+                    Import Data</li>
+                    <li>Desktop client import</li>
+                    <li>In-place import using OMERO.CLI</li>
```
Comment (mtbc) [inline events/13th-annual-users-meeting-2018.html]: OMERO.cli normally ?

File: events/13th-annual-users-meeting-2018.html [hunk 3]
```diff
@@ -229,6 +229,164 @@ <h2 class="event-day">Day 2 — Thursday 31st May, 2018 - Dalhousie Teaching Roo
...
+                <h5 class="event-time">10:00 - 11:00</h5>
+            </div>
+            <div class="small-12 medium-10 columns">
+                <h3 class="event-title">Workshop</h3>
+                <ul class="event-workshop">
+                    <li class="event-workshop-title">
+                    <span class="event-workshop-room">Room 1</span> -
+                    Raw Data analysis</li>
+                    <li>Fiji manual and scripting workflow</li>
+                    <li>Analysis workflow server side using OMERO.scripts</li>
```
Comment (mtbc) [inline events/13th-annual-users-meeting-2018.html]: consider hyphenating server-side

**Comment by jburel** [comment]
just an idea while driving back to get kids
for metadata workshop: ``The journey of metadata``

**Pull request merged**